### DISCUSSION TITLE
:set langmap in Helix?

### DISCUSSION THREAD

**Discussion by glow12121** [helix-editor/helix#5762] [Q&A] [answered]
According to [one Vim manual](https://r-notes.ru/49-knigi/spravochnik-po-vim/175-podderzhka-russkogo-yazyka-i-rusifikatsiya.html#russian-keymap), to avoid the need to switch keymaps in Normal mode (for example, when you have a Latin and a Cyrillic keyboard layouts), one needs to set this `langmap` option:

```
:set langmap=ФИСВУАПРШОЛДЬТЩЗЙКЫЕГМЦЧНЯ;ABCDEFGHIJKLMNOPQRSTUVWXYZ,фисвуапршолдьтщзйкыегмцчня;abcdefghijklmnopqrstuvwxyz
```

Is there something similar in Helix?

**Comment by CptPotato** [comment, marked as answer]
It is not implemented yet, but there's an open PR: #5046

**Reply by glow12121** [reply]
Thank you. Can there be some kind of workaround while this hasn't been implemented? Writing in markdown in Cyrillic keymap is too difficult when all hotkeys are in Latin letters only.

**Reply by CptPotato** [reply]
Do you have the necessary tools installed to build helix from source? If you want to give the feature in that PR a spin, you can fork the helix repository, merge the PR and build it yourself. But note that the pull request has some merge conflicts that you'd have to resolve first.

Let me know if you need some assistance with this.

**Reply by glow12121** [reply]
That's really beyond my competence but still thank you for letting me know.
